app/views/teaching_events/index.html.erb

```diff
@@ -2,30 +2,10 @@
   <header>
     <h1 class="heading-l heading--box-yellow">Events</h1>
 
-    <p>
-      Find out more about getting into teaching at a free event where you can get all your questions answered.
-    </p>
+    <p>Get expert advice, meet training providers, and talk to teachers at Get into Teaching events.</p>
```

Comment:
```suggestion
    <p>Get expert advice, meet training providers, and talk to teachers at Get Into Teaching events.</p>
```

Reply:
If the whole purpose of this text is to clarify the different event types, I wonder if we need to be a bit clearer about it? e.g. something more like:

> Choose from 2 types of free events: 
> 
> **Get Into Teaching events** where you can talk to expert advisers, a range of local training providers and current and former teachers. 
> 
> **Teacher training provider events** where specific training providers will talk about the courses they offer.

But then this does bulk up the text again, meaning the filters aren't as high up the screen (but if one of the filters is to choose an event type, and users are skipping past the info that explains this, then that feels a bit pointless).

@gemmadallmandfe @philspray

Reply:
Think this makes sense - and good to distinguish between multiple providers at GIT events vs one at provider events?

NB filter calls them 'Training provider events' so worth leading with that for consistency?

Choose from 2 types of free events:

**Get Into Teaching events** where you can talk to expert advisers, a range of local training providers and current and former teachers.

**Training provider events** where specific teacher training providers will talk about the courses they offer.